inference-engine/tests/functional/shared_test_classes/src/single_layer/memory.cpp

```diff
@@ -67,12 +67,13 @@ namespace LayerTestsDefinitions {
 
```

Comment:
and what about `TensorIterator`, `Loop`, `GRU\RNN\LSTMCells`, `GRU\RNN\LSTMSequence`?

Reply:
Analyzed, ran tests with these patterns, the function does not change with either of them